README.md

```diff
@@ -88,66 +111,45 @@ Working:
-  - Note that not all possible permutations of table names have been tested.
-    - i.e. there may be concerns around casing
-- MySql: Not Implemented yet
+## Performance
 
-# Performance
-
-Tests based on i-7 8750H, 32GB Ram, 2TB SSD, Windows 10 version Version 10.0.19041.630.
+Tests based on AMD Ryzen 9 3900X, 32GB Ram, Windows 10 Version 22H2.
 Databases running on Docker WSL2.
 
 All numbers are in msg/sec.
 
-|Test |SqlServer (normal)       |  SqlServer Batching     | Linq2Db     |vs Normal| vs Batching|
-|:-------------  |:------------- | :----------: | -----------: | -----------: | -----------: |
-|Persist |  164|427| 235|143.29%|55.04%|
-|PersistAll |  782|875| 5609|717.26%|641.03%|
-|PersistAsync |  630|846| 16099|2555.40%|1902.96%|
-|PersistAllAsync |  2095|902| 15681|748.50%|1738.47%|
-|PersistGroup10 |  590|680| 1069|181.19%|157.21%|
-|PersistGroup100 |  607|965| 5537|912.19%|573.78%|
-|PersistGroup200 |  628|1356| 7966|1268.47%|587.46%|
-|PersistGroup25 |  629|675| 2189|348.01%|324.30%|
-|PersistGroup400 |  612|1011| 7237|1182.52%|715.83%|
-|PersistGroup50 |  612|654| 3867|631.86%|591.28%|
-|Recovering |  41903|38766| 42592|101.64%|109.87%|
-|Recovering8 |  75466|65515| 63960|84.75%|97.63%|
-|RecoveringFour |  59259|51355| 58437|98.61%|113.79%|
-|RecoveringTwo |  41745|35512| 41108|98.47%|115.76%|
+| Test            |          SqlServer | SqlServer Batching | Linq2Db | vs Normal | vs Batching |
+|:----------------|-------------------:|-------------------:|--------:|----------:|------------:|
+| Persist         |                304 |                299 |     496 |   163.16% |     165.89% |
+| PersistAll      |               1139 |               1275 |    7893 |   692.98% |     619.06% |
+| PersistAsync    |               1021 |               1371 |   31813 |  3115.87% |    2320.42% |
+| PersistAllAsync |               2828 |               1395 |   29634 |  1047.88% |    2124.30% |
+| PersistGroup10  |                986 |               1034 |    1675 |   169.88% |     161.99% |
+| PersistGroup100 |               1054 |               1304 |    6249 |   592.88% |     479.22% |
+| PersistGroup200 |                990 |               1662 |    8086 |   816.77% |     486.52% |
+| PersistGroup25  |               1034 |               1010 |    3054 |   295.36% |     302.38% |
+| PersistGroup400 |               1049 |               2113 |    7237 |   689.59% |     342.50% |
+| PersistGroup50  |                971 |                980 |    4932 |   507.93% |     503.27% |
+| Recovering      |              60516 |              77688 |   64457 |   106.51% |      82.96% |
+| Recovering8     |             116401 |             101549 |  103463 |    88.89% |     101.88% |
+| RecoveringFour  |              86107 |              73218 |   66512 |    77.24% |      90.84% |
+| RecoveringTwo   |              60730 |              53062 |   43325 |    71.34% |      81.65% |
 
 ## Sql.Common Compatibility modes
```

Comment:
I'd say "Akka.Persistence.Sql.Common` here, just to be explicit.

Reply:
It is measured using SqlServer though, we should consider that as a possible variable, there might be optimization at the SQL generator level that might affect benchmarks

Reply:
My comment pertains to the "Sql.Common Compatibility modes" header, not the benchmarks